app/components/organization-settings.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  membership: Ember.computed('organization.organizationMemberships', 'session.currentUser', function() {
+    let user = this.get('session.currentUser');
+    if (user) {
+      let memberships = this.get('organization.organizationMemberships');
+      let membership = memberships.find(this.findMembership.bind(this));
+      console.log(membership);
+      return membership;
+    }
+  }),
+
+  findMembership: function(item, index, enumerable) {
+    let itemMemberId = item.belongsTo('member').id()
+    let itemOrganizationId = item.belongsTo('organization').id()
+    let currentUserId = this.get('session.currentUser.id');
+    let organizationId = this.get('organization.id');
+    return (itemMemberId === currentUserId) && (itemOrganizationId === organizationId);
+  },
```

Comment:
I'm repeating myself a lot with this and would love to find a better way to do this. It seems like the general idea here should be possible to extract out, though not sure how yet. Had enough trouble just getting it to work by tying it with yarn.